components/ui/Contact.tsx

```diff
@@ -7,11 +7,53 @@ export interface ContactProps {
   id: number;
   name: string;
   lastMessageAt: string;
+  Messages: [
+    {
+      id: number;
+      body: string;
+    }
+  ];
```

Comment:
```suggestion
  lastMessage: string
```

Just pass the last message rather than all messages

Reply:
On removing I got deep Scan issues.

Reply:
I don't understand how changing props defination is related to deepscan issue. Please check what the deepscan issue is saying and resolve appropriately.